eng/common/scripts/Helpers/DevOps-WorkItem-Helpers.ps1

```diff
@@ -180,6 +182,56 @@ function FindParentWorkItem($serviceName, $packageDisplayName, $outputCommand =
   return $null
 }
 
+$releasePlanWorkItems = @{}
+function FindReleasePlanWorkItem($serviceName, $packageDisplayName, $outputCommand = $false, $ignoreReleasePlannerTests = $true)
+{
+  $key = BuildHashKey $serviceName $packageDisplayName
+  if ($key -and $releasePlanWorkItems.ContainsKey($key)) {
+    return $releasePlanWorkItems[$key]
+  }
+
+  if ($serviceName) {
+    $condition = "[ServiceName] = '${serviceName}'"
+    if ($packageDisplayName) {
+      $condition += " AND [PackageDisplayName] = '${packageDisplayName}'"
+    }
+    else {
+      $condition += " AND [PackageDisplayName] = ''"
+    }
+  }
+  else {
+    $condition = "[ServiceName] <> ''"
+  }
+  $condition += " AND [System.State] <> 'Finished'"
+  if($ignoreReleasePlannerTests){
+    $condition += " AND [Tags] NOT CONTAINS 'Release Planner App Test'"
```

Comment:
is there a differente between `NOT CONTAINS` and `<>` ?

Reply:
One is string not equal and the other one is to check if it is in the list